### DISCUSSION TITLE
Longpress to switch between brightness, colour temp and rgb colour?

### DISCUSSION THREAD

**Discussion by smartroad** [nicufarmache/lovelace-big-slider-card#13] [Ideas]
Would it be possible to have it when the card is long pressed it would switch between setting the brightness, colour temp and rgb colour? (assuming available for the lamp)

**Comment by nicufarmache** [comment]
I was thinking of implementing something like this but I haven't yet decided which would be the right way to do it.  

If i just use the long press to switch between the attributes being changed it could become a little unclear which is the current active attribute (maybe it needs some sort of visual indication)

I have to think about it a little more.

**Reply by smartroad** [reply]
I don't know if it is possible, but maybe the background could change from the current for brightness to fading from light blue to yellow-ish for colour temp and the rainbow look for RGB?

**Comment by jazzyisj** [comment]
Long press brings up the more-info dialogue which gives you convenient access to all of those things.  Why reinvent the wheel?  
I certainly wouldn't want to lose access to that dialogue!

**Reply by smartroad** [reply]
Most wigdets have an option to say how they respond to tap and long tap in their settings, could just be an option depending on preference?
